Documentation/development-flow.md

```diff
@@ -0,0 +1,116 @@
+# Prerequisites
+1. GO installed
```

Comment:
any particular version required?  a link to golang install instructions would be nice too.